src/main/java/frc/robot/Constants.java

```diff
@@ -345,6 +345,8 @@ public static final class AutoConstants {
     public static final double kStationSlotSpacing = 0.202; // Distance between slots.
 
     public static final double kProcessorXOffset = -(0.222 + kFrameToRobotCenter); // TODO: Tune.
+
+    public static final double kAlgaeIntakePositionTwo = 0.1; //TODO: Tune.
```

Comment:
Let's name this in a way that indicates it's an x offset, e.g. `kAlgaeIntakeXOffset`.